documentation/index.md

```diff
@@ -40,18 +40,30 @@ If you are using [CocoaPods](https://cocoapods.org), then follow these [alternat
 ### 3. Segue for security concerns
 
 * Since Sparkle is downloading executable code to your users' systems, you must be very careful about security.
-* To let Sparkle be sure an update came from you (instead of a malicious attacker), you must do one of two things:
-  * If you're updating a regular app (not preference pane or plugin), then sign your updates via Apple's Developer ID program or your own certificate. Sparkle will ensure the new version's author matches the old version's. This is the recommended solution, as you don't need to manage DSA keys this way. This feature doesn't work with .pkg-based updates or binary-delta updates: for those, you'll have to use the signatures described below.
-    * Note that embedding the Sparkle.framework into the bundle of a Developer ID application requires that you code-sign the framework with your Developer ID keys. Xcode should do this automatically if you let it "Code Sign on Copy" Sparkle's framework.
-    * You can diagnose code signing problems with [RB App Checker app](http://brockerhoff.net/RB/AppCheckerLite/) and by checking logs in the Console.app.
-  * If you can't code sign your app, you can include a [DSA signature](https://en.wikipedia.org/wiki/Digital_signature) of the SHA-1 hash of your published update file.
-    * First, make yourself a pair of DSA keys; Sparkle includes a tool to help:
-    * (from the Sparkle distribution root):<br />
-<code>./bin/generate_keys.sh</code>
-    * You can use the keys this tool generates to sign your updates.
-    * Back up your private key (dsa_priv.pem) and <strong>keep it safe.</strong> You don't want anyone else getting it, and if you lose it, you won't be able to issue any new updates.
-    * Add your public key (dsa_pub.pem) to the Resources folder of your Xcode project.
-    * Add a <code>SUPublicDSAKeyFile</code> key to your Info.plist; set its value to your public key's filename—unless you renamed it, this will be dsa_pub.pem.
+* To let Sparkle be sure an update after being downloaded is not corrupted and came from you (instead of a malicious attacker), we recommend:
```

Comment:
This is grammatically unusual. Perhaps:

> To let Sparkle know that a downloaded update is not corrupted and came from you (instead of a malicious attacker), we recommend: